src/main/java/com/jlr/ttl/ds/api/controllers/VehicleController.java

```diff
@@ -40,18 +41,35 @@ public ResponseEntity<DSResponse<List<VehicleResponse>>> getAllVehicles() {
                 .build(), HttpStatus.OK);
     }
 
-    @GetMapping("/by_id/{id}")
-    public ResponseEntity<DSResponse<VehicleResponse>> getVehicleById(@PathVariable(value = "id") String id){
+    /**
```

Comment:
When refactoring the info variable do not forget to update the documentation

Reply:
Documentation has been updated